strawberryfields/gbs/vibronic.py

```diff
@@ -148,24 +166,29 @@ def gbs_params(
     >>> p = gbs_params(w, wp, Ud, d)
 
     Args:
-        w (array): normal mode frequencies of the electronic ground state (:math:`\text{cm}^{-1}`)
-        wp (array): normal mode frequencies of the electronic excited state (:math:`\text{cm}^{-1}`)
+        w (array): normal mode frequencies of the initial electronic state (:math:`\mbox{cm}^{-1}`)
+        wp (array): normal mode frequencies of the final electronic state (:math:`\mbox{cm}^{-1}`)
         Ud (array): Duschinsky matrix
         d (array): Duschinsky displacement vector corrected with wp
+        T (float): temperature (Kelvin)
 
     Returns:
-        tuple[array, array, array, array]: the first interferometer unitary matrix :math:`U_{1}`,
-        the squeezing parameters :math:`r`, the second interferometer unitary matrix :math:`U_{2}`,
-        and finally the displacement parameters :math:`\alpha`
+        tuple[array, array, array, array, array]: the two-mode squeezing parameters :math:`t`,
+        the first interferometer unitary matrix :math:`U_{1}`, the squeezing parameters
+        :math:`r`, the second interferometer unitary matrix :math:`U_{2}`, and the displacement
+        parameters :math:`\alpha`
     """
-    Wi = np.diag(w ** -0.5)
-    Wp = np.diag(wp ** 0.5)
-    J = Wp @ Ud @ Wi
-
-    U2, s, U1 = np.linalg.svd(J)
+    if T < 0:
+        raise ValueError("Temperature must be zero or positive")
+    if T > 0:
```

Comment:
I wonder if we get any errors if a user inputs an extremely low temperature, for example `T=1e-20` ?

Reply:
No.